x-pack/test/alerting_api_integration/security_and_spaces/group2/tests/alerting/alerts.ts

```diff
@@ -1290,6 +1294,69 @@ instanceStateValue: true
           }
         });
 
+        it('should filter alerts by kql', async () => {
```

Comment:
I think we could also add a functional test for the timestamp filter using the current time to create a rule that includes alerts from the current time and excludes alerts from the current time, WDYT?

Reply:
Ok added a test for `hours` too.